package.xml

```diff
@@ -21,6 +22,8 @@
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
 
+  <member_of_group>rosidl_interface_packages</member_of_group>
```

Comment:
I'm wondering whether I should insist on this, but - it is normally best practice to separate ROS interfaces into a standalone package without any other functionality (`topic_tools_interfaces` or `topic_tools_msgs` name maybe). This is done so that an external "caller" application doesn't actually need to depend on all the resulting binaries of of this package, only on the msg/srv definitions as a much smaller dependency. `topic_tools` would also depend on this new package.

Again, "normally best practice" doesn't mean it's a hard line. How do you feel about that change? The interfaces packages would live in this same repository, it would just have its own package.xml in a subdirectory. It's probably not a very big deal either way.

Reply:
I agree with separating the packages, since I just added the interface into the same package in accordance with the implementation of `ros1`.

Reply:
addressed at https://github.com/ros-tooling/topic_tools/pull/26/commits/52db593c491f42efb2ea513bf076bcd38fc117aa
```suggestion
  <member_of_group>rosidl_interface_packages</member_of_group>
```
```suggestion
  <member_of_group>rosidl_interface_packages</member_of_group>
```

Reply:
addressed at https://github.com/ros-tooling/topic_tools/pull/26/commits/52db593c491f42efb2ea513bf076bcd38fc117aa